src/main/resources/assets/scicraft/models/item/downquark_quantumfield.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parent": "scicraft:block/downquark_quantumfield",
+  "display": {
+    "thirdperson": {
```

Comment:
Is there a reason to deviate from the default Minecraft behavior here? Or is this residual code from MCreator maybe?

Reply:
Actually it doesn't deviate from default vanilla Minecraft behavior. I used the models and textures from Elementarycraft, I just renamed some textures and put them in a better file structure. Since it had the special rotation, translation and scale I expected it to deviate from vanilla behavior and kept it to be consistent with what was already in Elementarycraft.